backend/gce.go

```diff
@@ -1029,8 +1049,31 @@ func (p *gceProvider) stepInsertInstance(c *gceStartContext) multistep.StepActio
 		}
 	}
 
-	p.apiRateLimit(c.ctx)
-	op, err := p.client.Instances.Insert(p.projectID, c.zoneName, inst).Context(c.ctx).Do()
+	b := backoff.NewExponentialBackOff()
+	b.InitialInterval = 1 * time.Second
+	b.MaxElapsedTime = 1 * time.Minute
+
+	err = backoff.Retry(func() error {
+		p.apiRateLimit(c.ctx)
+		op, err := p.client.Instances.Insert(p.projectID, c.zoneName, inst).Context(c.ctx).Do()
+		if err != nil {
+			if !c.zonePinned {
```

Comment:
I'm curious: is it most likely that an error here is because of an issue with the zone? Do we expect that switching to another will usually fix our issue? Is there anything we could look for in the error to determine if that makes sense?

Reply:
It's one of the top 3 errors, and I couldn't think of a reason why switching zones/retrying would be a bad idea. I'm hesitant to get more specific based on what's in the error given we don't have error types on which we can rely.

Reply:
Ok, that makes sense! 👍